zstacklib/zstacklib/utils/lv_metadata.py

```
@@ -255,6 +255,54 @@ def _do_cleanup(self, metadataPath):
+            metadata_lvs = scan_metadata_lvs(vg_uuid, lvm.list_lvs)
+        except Exception as e:
+            logger.warn("scan_metadata_lvs failed on vg %s: %s", vg_uuid, e)
+            return {'cleanedCount': 0, 'failedCount': 0, 'error': str(e)}
+
+        for item in metadata_lvs:
+            lv_path = item.get('lv_path')
+            try:
+                if lvm.lv_exists(lv_path):
+                    delete_metadata_lv(lv_path, lvm.delete_lv)
+                cleaned += 1
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)